core/ExecutePipe.cpp

```diff
@@ -69,35 +69,43 @@ namespace olympia
     {
         // FIXME: Now every source operand should be ready
         const auto & src_bits = ex_inst->getSrcRegisterBitMask(reg_file_);
-        sparta_assert(scoreboard_views_[reg_file_]->isSet(src_bits),
+        if(scoreboard_views_[reg_file_]->isSet(src_bits)){
+            sparta_assert(scoreboard_views_[reg_file_]->isSet(src_bits),
                       "Should be all ready source operands ... " << ex_inst);
```

Comment:
Probably can remove this assert.

Reply:
Addressed in [4fd928d](https://github.com/riscv-software-src/riscv-perf-model/pull/51/commits/4fd928d4400af49de00ac3f5f8b5c4990bdbb2fd)